CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+### 5.1.2
+
+* Fix broken RSpec split by test examples feature when `SPEC_OPTS` is set in Queue Mode
```

Comment:
Have you considered adding a brief explanation on how the fix works? (e.g. ignore `SPEC_OPTS`, ensure formatter)

Reply:
ok. Updated the text.